plugins/headlamp-plugin/template/src/storybook.test.tsx

```diff
@@ -1,5 +0,0 @@
-// @ts-nocheck
-import { initTests } from '@kinvolk/headlamp-plugin/config/storyshots/storyshots-test';
```

Comment:
Are you sure this can be removed?

Reply:
since it's a template for new plugins so I don't see a risk there. If storyshots plugin is deprecated I think it's fair for us to not support it. Keeping storyshots tests for plugins is not trivial, we would effectively need to maintain a copy of 'storyshots-tests' plugin that works with vite